nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -85,7 +85,19 @@ class BtServiceNode : public BT::SyncActionNode
   {
     on_tick();
     auto future_result = service_client_->async_send_request(request_);
+    return check_future(future_result);
+  }
+
+  // Fill in service request with information if necessary
+  virtual void on_tick()
+  {
+    request_ = std::make_shared<typename ServiceT::Request>();
+  }
 
+  // Check the future and decide the status of Behaviortree
+  virtual BT::NodeStatus check_future(
+      std::shared_future<typename ServiceT::Response::SharedPtr> future_result)
```

Comment:
Linter error. Use ament_uncrustify to fix it.

Reply:
Fixed it. Thanks for the suggestion.